File: src/main/java/yaasl/server/persistence/CostSharingRepository.java
package yaasl.server.persistence;

import org.springframework.data.repository.CrudRepository;
import yaasl.server.model.CostSharing;
import yaasl.server.model.PilotRole;

public interface CostSharingRepository extends CrudRepository<CostSharing, Long> {

    CostSharing findByDescription(String description);

}

File: src/main/kotlin/yaasl/server/persistence/CostSharingRepository.kt
package yaasl.server.persistence

import org.springframework.data.repository.CrudRepository
import yaasl.server.model.CostSharing
import yaasl.server.model.PilotRole

interface CostSharingRepository : CrudRepository<CostSharing, Long> {

    fun findByDescription(description: String): CostSharing

}
